Compiler/NFFrontEnd/NFInstNode.mo

```diff
@@ -258,7 +259,8 @@ uniontype InstNode
 
   function scopePrefix
     input InstNode node;
-    input output Prefix prefix = Prefix.NO_PREFIX();
+    input output Prefix
+    prefix = Prefix.NO_PREFIX();
```

Comment:
No way, that seems to be like a stray enter, probably still getting used with this new HP keyboard :)